__init__.py

```diff
@@ -441,9 +441,9 @@ class ThangsLink(bpy.types.Operator):
     bl_label = "Redirect to Thangs"
 
     def execute(self, context):
-        amplitude.send_amplitude_event("nav to thangs", event_properties={})
-        webbrowser.open(thangs_config.thangs_config["url"] + "search/" + fetcher.query +
-                        "?utm_source=blender&utm_medium=referral&utm_campaign=blender_extender", new=0, autoraise=True)
+        amplitude.send_amplitude_event("Thangs Blender Addon - Nav to Thangs", event_properties={})
+        webbrowser.open(thangs_config.thangs_config["url"] + "search/" + str(fetcher.searchTerm) +

```

Comment:
should urlencode `fetcher.searchTerm` unless that's done already

Reply:
+1.  Since we're opening a browser it seems to already deal with this in _some_ cases (e.g. do a search for "joe mama" and the browser will automatically encode the space), but not others (e.g. do a search for "joe/mama")

[Url encoding](https://stackoverflow.com/a/54135995/8802) the term would fix both of those cases